test/parallel/test-trace-events-category-used.js

```diff
@@ -35,6 +36,7 @@ procEnabled.once('exit', common.mustCall(() => {
 const procDisabled = cp.spawn(
   process.execPath,
   [ '--trace-event-categories', 'other',
+    '--no-warnings',
```

Comment:
Why this?

Reply:
just to make the test less noisy since the import of `internal/test/binding` emits a warning